roles/common/tasks/networking.yml

```diff
@@ -1,16 +1,67 @@
 ---
-- name: network interfaces file
-  copy: content="{{ network_interfaces }}" dest=/etc/network/interfaces
-        force=yes owner=root group=root mode=0644
-  when: network_interfaces is defined
-  tags: ['prereboot']
+- block:
+  - name: set network config dir path - ubuntu
+    set_fact:
+      network_dir_path: /etc/network/interfaces.d
+    when: os == 'ubuntu'
```

Comment:
This is tied to this card. https://trello.com/c/Mrv3ZTGc/87-use-ansible-os-family-instead-of-os
This should be fixed when a solution is decided for this task.